kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/PtvServiceProvider.java

```diff
@@ -163,48 +181,68 @@ private Service transform(IVmOpenApiService ptvService) {
     
     return service;
   }
-  
-  private List<LocalizedValue> getLocalizedValues(String type, List<VmOpenApiLocalizedListItem> items) {
-    if (items != null && !items.isEmpty()) {
-      List<LocalizedValue> result = new ArrayList<>();
-      List<VmOpenApiLocalizedListItem> typeItems = new ArrayList<>();
-      
-      for (VmOpenApiLocalizedListItem item : items) {
-        if (type.equals(item.getType())) {
-          typeItems.add(item);
+
+  private void parseChannelIds(IVmOpenApiService ptvService, Service service) {
+    List<String> electronicChannelIds = new ArrayList<>();
+    List<String> phoneChannelIds = new ArrayList<>();
+    List<String> printableFormChannelIds = new ArrayList<>();
+    List<String> webpageChannelIds = new ArrayList<>();
+    List<String> serviceLocationChannelIds = new ArrayList<>();
+    /**
+    
+    FIXME: Temporarity disabled because PTV does not respect design REST principles. Currently trying 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1134)